### PR TITLE
remove iframe border in PosterHall

### PR DESCRIPTION
**Pull request by yarikoptic**
- IMHO it makes display more "coherent" (there are no borders around other
  elements, e.g. drop down Schedule, Chat, etc)
- make it possible for presented in iframe material to "meld" into the page
  better, again making it more "coherent" with the rest
  - opens possibility to take advantage of that e.g. with black background
    so presented stuff is nicely integrated into the display instead of aleniated
    with the border

### CODE REVIEW

File: src/components/templates/PosterPage/PosterPage.tsx [hunk 1]
```diff
@@ -183,6 +183,7 @@ export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
             title={title}
             allow={IFRAME_ALLOW}
             allowFullScreen
+            frameBorder="0"
```
Comment (0xdevalias) [inline src/components/templates/PosterPage/PosterPage.tsx:186]: I’m not 100% off the top of my head but I’m pretty sure this is a legacy prop and this should be done with CSS these days.

**Review by 0xdevalias — CHANGES_REQUESTED** [review]

File: src/components/templates/PosterPage/PosterPage.tsx (hunk 1 above)
Comment (yarikoptic) [reply src/components/templates/PosterPage/PosterPage.tsx]: done! commit was rewritten and force pushed

**Comment by 0xdevalias** [comment]
@mike-lvov and/or @sparkletown/product You’re more versed in the PosterHall designs. Is removing the border like this fine?

@yarikoptic can you please provide before/after screenshots to make comparison/decision easier.

File: src/components/templates/PosterPage/PosterPage.tsx (hunk 1 above)
Comment (0xdevalias) [reply src/components/templates/PosterPage/PosterPage.tsx]: Thanks :)

**Comment by yarikoptic** [comment]
> @yarikoptic can you please provide before/after screenshots to make comparison/decision easier.

hm... have I spoiled someone? ;)

before | after
--- | ---
image| image

**Comment by yarikoptic** [comment]
Please let me know if I could facilitate the review of this non elaborate PR.

**Review by mike-lvov — APPROVED** [review]
LGTM

**Comment by yarikoptic** [comment]
oh, it was for staging, not ohbm, oh well -- may be would still percolate ;)  Thank you @mike-lvov !

**Comment by 0xdevalias** [comment]
@yarikoptic 
image

**Pull request merged**